micropub.php

```diff
@@ -122,7 +120,7 @@ public static function parse_query( $wp ) {
 		static::load_input();
 		if ( WP_DEBUG ) {
 			error_log( 'Micropub Data: ' . serialize( $_GET ) . ' ' .
-					   serialize( static::$input ) );
+			serialize( static::$input ) );
```

Comment:
please also revert these changes that remove indentation from line continuations. the indentation is needed for readability, and both wordpress's and PHP's code standards agree:

https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#indentation
http://pear.php.net/manual/en/standards.funcalls.php#standards.funcalls.splitfunccall

again, not sure what PHPCS is smoking. :P